File: pgm/ABC_Java/ABC_227_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int zero = 0;
        final int one = 1;
        final int N = Integer.parseInt (arr[0]);
        final int K = Integer.parseInt (arr[1]);
        final int A = Integer.parseInt (arr[2]);
        System.out.println ((K + A - one) % N == zero ? N : (K + A - one) % N);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_227_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val space: String = " "
const val one: Int = 1
const val zero: Int = 0

fun main() {
    var (N: Int, K: Int, A: Int) = readln().split(space).map { it.toInt() }
    if ((K + A - one) % N == zero) {
        println(N)
    } else {
        println((K + A - one) % N)
    }
    kotlin.system.exitProcess(0)
}